src/mm-datepicker/mm-datepicker.js

```diff
@@ -167,6 +162,19 @@
 					});
 				}
 			}
+
+			if (this.areDatesValid() && this.useRange && this._rangePresets) {
+				var found = false;
+				this._rangePresets.forEach(function(preset) {
+					if(preset.range.start.isSame(sd, "day") && preset.range.end.isSame(ed.add(1,"second"),"day")) {
+						this.rangeValue = String(preset.value);
+						found = true;
+					}
+				},this);
+				if (!found) {
+					this.rangeValue = -1;
```

Comment:
Can we use empty-string ("") instead of -1 here to keep the types more consistent?